scripts/prune_workers.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+# Log in to Concourse
+fly -t networking-extensions login -c https://concourse.arp.cloudfoundry.org/ -u admin_concourse -p `grep CONCOURSE_ADMIN_PWD /concourse/.concourse.env | cut -d = -f 2`
```

Comment:
minor style thing. The `$(subprocess)` call is a bit easier to see than just backticks.

A separate variable instead of inline resolution of a multi-step pipeline is also easier to read. And finally, the quotes around variables are always a good idea / good habit to avoid any funky business with anything potentially containing whitespace.

```suggestion
CONCOURSE_PASSWORD=$(grep CONCOURSE_ADMIN_PWD /concourse/.concourse.env | cut -d = -f 2)
fly -t networking-extensions login -c https://concourse.arp.cloudfoundry.org/ -u admin_concourse -p "$CONCOURSE_PASSWORD"
```